utils.py

```diff
@@ -259,3 +259,24 @@ def print_opts(flats):
             for k, v in flats.items()
         )
     )
+
+def get_model_list(dirname, key):
+    """get last model in dirname, whose name contain key
+    Arguments:
+        dirname {str} -- directory name
+        key {str} -- "key" in the model name
+    Returns:
+        last_model_name {str} -- last model name
+    """
+    if os.path.exists(dirname) is False:
+        return None
+    gen_models = [
+        os.path.join(dirname, f)
+        for f in os.listdir(dirname)
+        if os.path.isfile(os.path.join(dirname, f)) and key in f and ".pth" in f
+    ]
+    if gen_models is None:
```

Comment:
gen_models can't be `None` but it can be empty

Reply:
Ah this was from the master, I hadn't noticed.

Reply:
Actually this function is not used in the current way of resuming training. I will delete it.

Reply:
I ended up keeping it, just in case I change the resume training function later